src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -343,7 +343,12 @@ private Task<DocumentDatabase> CreateDatabase(StringSegment databaseName, bool i
         {
             var config = CreateDatabaseConfiguration(databaseName, ignoreDisabledDatabase);
             if (config == null)
-                return null;
+            {
```

Comment:
This is a good change, yes. Note that you can also make this using `Task.FromResult(null)`